src/components/NetworkNodeSelector/NetworkNodeSelectorTs.ts

```diff
@@ -148,6 +150,13 @@ export class NetworkNodeSelectorTs extends Vue {
         this.$emit('input', { nodePublicKey: '' });
     }
 
+    public handleSelectCustomNode() {
+        if (this.customNode !== '') {
+            Vue.set(this, 'formNodeUrl', this.customNode);
+            Vue.set(this, 'customNode', '');
```

Comment:
@james-lin71 I believe you need to call fetchNodePublicKey() here to make sure the entered node is valid

Reply:
Updated!